### DISCUSSION TITLE
How to code this rhythm in Hydrogen AVL Drum Kit

### DISCUSSION THREAD

**Discussion by vinodpkd** [hydrogen-music/hydrogen#2020] [Hydrogen on Windows]
Hi 
I have a question here.
Ref : https://musescore.com/user/35273741/scores/6864978, 4th measure.
![image](https://github.com/user-attachments/assets/475b81eb-80b2-49fa-890a-32a1ea1fb796)
bpm - 134.5
How to code the above rhythm in H2.
My try:
![image](https://github.com/user-attachments/assets/1c440e19-daa3-4022-b286-3d9d9c08f536)
![image](https://github.com/user-attachments/assets/57ba7332-79ca-41c7-ad26-2f31a086f295)

I am not getting the timings right. Its not matching with the sound I am hearing in musescore.

**Comment by BigAlbertMusic** [comment]
From the third time it seems the second snare have to go more to the right and time 4 the last note have to go right too.

**Comment by cme** [comment]
Not that I understand stave at all, but for the third beam, on snare, you have:

```
|3       |
|O-OO--O-|
```

but my (admittedly awful) understanding of stave would be that this should correspond to

```
|3       |
|O---O-O-|
```

**Comment by NoAliby** [comment]
Like @BigAlbertMusic said the second snare is a grace note to the third (or rather, as a grace note has no "size", the Second Snare beat is a flam). Add this in Hydrogen by switching to a 64th note resolution and add it just before the Beat on 3, where I have drawn a red down arrow.

@cme is correct. You have an extra note in the third bar. Should only be a quarter note and then two 1/8th notes. 
MuzeScore shows the 2nd note (Circled) as a sudo-grace note with a smaller body, but it is just a standard 1/8th note

![Untitled](https://github.com/user-attachments/assets/72711c4d-ad2b-490b-8a42-89db6a8f70f3)
